### ISSUE TITLE
0.8.0 doesn't recognize variable names properly

### ISSUE THREAD

**Issue by Tableau-David-Potter** [realm/SwiftLint#522]
I just installed 0.8.0 and now SwiftLint is giving me an error on every variable name.  For example:

```swift
    public func generateError(parameters parameters: [CVarArgType]) -> NSError {

        let error = self.generateError(format: self.localizedDescription, parameters: parameters)
        return error
    }
```

SwiftLint gives me the following error on the first line above at the first instance of `parameters`:

> Variable Name Violation: Variable name should be between 3 and 45 characters long: 'parameters' (variable_name)

**Background**
- Installed using `brew install swiftlint`
- I'm using a config file that sets the `max_length` of `variable_name` to 45.
- This didn't happen with 0.7.2.

**Comment by Tableau-David-Potter** [comment]
Oh, one more thing.  Apparently there is no way to revert to a specific version of swiftlint using brew.  Not sure how swiftlint is different than many other tools.

**Comment by jpsim** [comment]
> Oh, one more thing. Apparently there is no way to revert to a specific version of swiftlint using brew. Not sure how swiftlint is different than many other tools.

That's a Homebrew limitation.

I'm looking into your first issue now.

**Comment by jpsim** [comment]
I just merged a fix to `master`. Thanks for reporting this!

**Comment by Tableau-David-Potter** [comment]
Thanks.  When do you think it will be available via homebrew?

**Comment by jpsim** [comment]
I don't know when we'll be cutting the next release.
